Libplanet/Tx/Transaction.cs

```diff
@@ -614,7 +619,7 @@ public void Validate()
                 throw new InvalidTxSignatureException(Id, message);
             }
 
-            if (!new Address(PublicKey).Equals(Signer))
+            if (!PublicKey.ToAddress().Equals(Signer))
```

Comment:
Does this change any behavior?

Reply:
Yeah, I thought it's better to read than `new Address(PublicKey)`. But now, there seems to be no difference. I will revert it.

Reply:
It's a subject matter, but I just though it's irrelevant to the purpose of the commit.

Reply:
Okay, I reverted it.